featurebyte/models/feature.py

```diff
@@ -44,18 +45,9 @@
     DefinitionHashExtractor,
     DefinitionHashOutput,
 )
-
-
-class EntityRelationshipInfo(FeatureByteBaseModel):
-    """
-    Schema for entity relationship information (subset of existing RelationshipInfo)
-    """
-
-    id: PydanticObjectId = Field(default_factory=ObjectId, alias="_id", allow_mutation=False)
-    relationship_type: RelationshipType
-    entity_id: PydanticObjectId
-    related_entity_id: PydanticObjectId
-    relation_table_id: PydanticObjectId
```

Comment:
Move this to `featurebyte.query_graph.model.entity_relationship_info` so that it can be imported inside `featurebyte.query_graph`.